src/python/pants/backend/project_info/tasks/export.py

```diff
@@ -104,11 +104,8 @@ def register_options(cls, register):
              help='Causes libraries with javadocs to be output.')
     register('--sources', type=bool,
              help='Causes sources to be output.')
-    # Required by IvyTaskMixin.
-    # TODO: Remove this once IvyTaskMixin registers an --ivy-jvm-options option.
-    # See also https://github.com/pantsbuild/pants/issues/3200.
-    register('--jvm-options', type=list, metavar='<option>...',
-             help='Run Ivy with these extra jvm options.')
```

Comment:
Does this need to be deprecated first?

Reply:
This option is not gone. It is just a bit funky now because 
`ExportTask -> CoursierMixin -> NailgunTask -> NailgunTaskBase -> NailgunTaskBase -> JvmToolMixin` which has `--jvm-options`, so this is transparent to user.